### PR TITLE
chore: content update on homepage about section to link to nft school

### LINKED ISSUE #481: Messaging on homepage around IPFS / Filecoin and why they're important to use for NFTs
Those landing on NFT.Storage without any background knowledge on content addressing might be confused why it's a valuable product. Think we cover this a bit, but think it can be improved in the `About` section.

Would make the text: `nft.storage is a brand new service, built specifically for storing off-chain NFT data. Data is stored decentralized on [IPFS](https://ipfs.io/) and [Filecoin](https://filecoin.io/), and are referenced using decentralized, content addressed IPFS URIs that can be used in your smart contract without fear of the data being referenced changing. To learn more about the importance of content addressing, check out this page on [NFT School](https://nftschool.dev/concepts/content-addressing/).`

### PR DESCRIPTION
**Pull request by zebateira**
### How it looks

image

Closes #481

### CODE REVIEW

**Comment by zebateira** [comment]
@dchoi27 do you want a line break after before the last sentence "To learn more..."?

File: packages/website/pages/index.js [hunk 1]
```diff
@@ -158,6 +158,16 @@ function About() {
           <a href="https://filecoin.io" className="black">
             Filecoin
           </a>
+          , and are referenced using decentralized, content addressed IPFS URIs
```
Comment (dchoi27) [inline packages/website/pages/index.js:161]: can we take out the word "decentralized" and add a hyphen to "content-addressed" here (so it just reads "referenced using content-addressed IPFS URIs)...")

File: packages/website/pages/index.js [hunk 2]
```diff
@@ -158,6 +158,16 @@ function About() {
...
+          that can be used in your smart contract without fear of the data being
+          referenced changing. To learn more about the importance of content
```
Comment (dchoi27) [inline packages/website/pages/index.js:163]: yeah adding a line break here sounds good

**Review by dchoi27 — CHANGES_REQUESTED** [review]

File: packages/website/pages/index.js (hunk 1 above)
Comment (zebateira) [reply packages/website/pages/index.js]: done 👍

**Comment by zebateira** [comment]
@dchoi27 updates done. LMK if everything is ok now

**Pull request merged**